src/icons/Arrow/Arrow.tsx

```diff
@@ -0,0 +1,19 @@
+const Arrow = () => (
```

Comment:
Don't do a folder for it
Just put all files to the `icons` folder
Also, give a name like `ArrowIcon`
```suggestion
const ArrowIcon = () => (
```
![image](https://github.com/fe-feb23-code-hunters/react_phone-catalog/assets/91411692/7cfe6ba7-ff16-4900-83bf-646589796ecb)